### DISCUSSION TITLE
query of result not friendly

### DISCUSSION THREAD

**Discussion by sytpb** [sidorares/node-mysql2#3088] [Q&A]
```sql
SELECT EXISTS (
    SELECT 1 
    FROM users 
    WHERE email = 'johndoe@example.com'
);
```

sometimes we query data in Db using Select Exists , hope just getting result YES or No ,  but the result we get like 

`[
  {

    'EXISTS (\n       SELECT EXISTS (
    SELECT 1 
    FROM users 
    WHERE email = 'johndoe@example.com'
))': 1

  }
]
`

we need to pick "1"  out ,a little hard and wierd.  so this is not friendly for client code .    hope that have some field like 'affectedrows':1 , or 'Exists': true/false. ...

Thanks

**Comment by sidorares** [comment]
why not just `SELECT count(*) as userCount FROM users WHERE email = 'johndoe@example.com'`?
